scripts/irods/controller.py

```diff
@@ -111,14 +111,14 @@ def start(self, write_to_stdout=False, test_mode=False):
 
         self.wait_for_server_to_start()
 
-    def stop(self, timeout=20, graceful=False):
+    def stop(self, graceful=False):
         l = logging.getLogger(__name__)
         self.config.clear_cache()
         l.debug('Calling stop on IrodsController')
 
         server_pid = self.get_server_pid()
         if server_pid is None:
-            l.info('iRODS server is not running')
+            l.info('No iRODS server running')
```

Comment:
this might affect a test somewhere... just keep an eye out

Reply:
I introduced the `is not running` phrase. I changed these lines to match what has been there since I joined.

I doubt this will break anything, but if it does, the tests will catch it.